report_templates/job_invocation_report.erb

```diff
@@ -0,0 +1,35 @@
+<%#
+name: Job Invocation report
```

Comment:
with the new nomenclature, this should now be "Jobs - Invocations", also please rename the file name